duckduckgo/iOS
# New header design: icons, app colors and edit text mode

**Task/Issue URLs**
Icons: https://app.asana.com/0/72649045549333/614787299259405
Colors: https://app.asana.com/0/72649045549333/614787299259403
SearchBar: https://app.asana.com/0/72649045549333/614787299259402

**Description:**
Adds new header design

**Steps to test this PR:**
Run the app and check that
• The icons and colors are updated
• The edit text mode hides other icons and shows new background when in edit mode

NOTE: There is a separate task to add a "count" to the tabs button.
